### ISSUE TITLE
Deprecated methods in ADAuthenticationViewController.m

### ISSUE THREAD

**Issue by paulw11** [AzureAD/azure-activedirectory-library-for-objc#1278]
This file generates two deprecation warnings.

 * `shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation)` - This can be replaced with something like:

```
- (UIInterfaceOrientation) preferredInterfaceOrientationForPresentation {
	if ( UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad ) {
		return [super preferredInterfaceOrientationForPresentation];
	} else {
		return UIInterfaceOrientationPortrait;
	}
	
}
```

* `viewDidUnload` 
The implementation of this function only performs debug logging.  It could be removed.

**Comment by oldalton** [comment]
This has been addressed on the dev branch already and will be included in the next release.

**Comment by oldalton** [comment]
Should be fixed in ADAL 2.7.5. Please try out.
